core/scripts/release/package-test.sh

```diff
@@ -20,7 +20,7 @@ cd /tmp/lighthouse-local-test
 
 npm init -y
 npm install "$LH_ROOT/lighthouse-$VERSION.tgz"
-npm install lighthouse-plugin-publisher-ads
```

Comment:
Could we add a `TODO(esmodules)` here for when this is out of beta. I feel like we'll forget.

Reply:
#14375